tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -326,6 +327,7 @@ def emit_view_lambda(f: NativeFunction, unpacked_bindings: List[Binding]) -> str
     updated_unpacked_args: List[str] = []
     known_view_arg_simple_types: List[CType] = [
         BaseCType(longT),
+        BaseCType(dimnameT),
```

Comment:
Had to add after adding `SymInt` to `select.int` in the yaml file. Couldn't compile otherwise.